atlp-rwanda/Team-Sostene-E-commerce-fe
# #185172118  Notifications

#### What does this PR do?
- Configure notifications
- Notification should be sent to the right user
- Notification Icon
#### Description of Task to be completed?
- the user should be notified of every single important event/action in the app
These are real-time notifications implemented with (socket.io)[https://socket.io/]
#### How should this be manually tested?
- A logged In user can see notification Icon with number of notifications
- Set the token in state. 
#### What are the relevant pivotal tracker/Trello stories?
#185172118
